.github/workflows/windows.yml

```diff
@@ -0,0 +1,58 @@
+name: github-windows
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${ {github.event_name }}-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{github.event_name == 'pull_request'}}
+
+jobs:
+  windows-threads:
+    name: Windows Threads
+    runs-on: windows-2022
+
+    env:
+      KOKKOS_INSTALL_DIR: C:\kokkos-install
+
+    steps:
+    - name: Install boost
+      shell: bash
+      run:
+        choco install boost-msvc-14.3
+    - uses: actions/checkout@v3
+      with:
+        repository: kokkos/kokkos
+        ref: 4.0.00
+        path: ${GITHUB_WORKSPACE}/../kokkos
+    - name: Install Kokkos
+      shell: bash
+      working-directory: ${GITHUB_WORKSPACE}/../kokkos
+      run: |
+        mkdir build
+        cd build
+        cmake -D CMAKE_INSTALL_PREFIX=${KOKKOS_INSTALL_DIR} \
+              -D Kokkos_ENABLE_THREADS=ON \
+              ..
+        cmake --build . --target install -- -m
+    - uses: actions/checkout@v3
+    - name: Configure ArborX
+      shell: bash
+      run: |
+        mkdir build
+        cd build
+        cmake \
+          -D CMAKE_BUILD_TYPE=Debug \
+          -D CMAKE_PREFIX_PATH="${KOKKOS_INSTALL_DIR}" \
+          -D ARBORX_ENABLE_MPI=OFF \
+          -D ARBORX_ENABLE_TESTS=ON \
+          -D ARBORX_ENABLE_EXAMPLES=ON \
+          -D ARBORX_ENABLE_BENCHMARKS=OFF \
```

Comment:
Why are the benchmarks off? Because `benchmark` not installed?

Reply:
Yes, I haven't bothered with benchmarks yet. Getting this to work should be enough initially and we can always enable more options.